src/screens/send/components/fee-advanced/fee-advanced.tsx

```diff
@@ -0,0 +1,103 @@
+
+        this.state = {
+            inputGasPrice: blockchainInstance.account
+                .convertToGasPriceUnit(new BigNumber(Number(props.gasPrice)))
+                .toString(),
+            inputGasLimit: props.gasLimit.toString()
+        };
+    }
+    public addGasPrice(value: string) {
+        const blockchainInstance = getBlockchain(this.props.blockchain);
+
+        this.setState({ inputGasPrice: value });
+        this.props.onInputFees(
+            blockchainInstance.account.convertFromGasPriceUnit(new BigNumber(Number(value))),
+            new BigNumber(Number(this.state.inputGasLimit))
+        );
+    }
+    public addGasLimit(value: string) {
+        const blockchainInstance = getBlockchain(this.props.blockchain);
+
+        this.setState({ inputGasLimit: value });
+        this.props.onInputFees(
+            blockchainInstance.account.convertFromGasPriceUnit(
+                new BigNumber(Number(this.state.inputGasPrice))
+            ),
+            new BigNumber(Number(value))
+        );
+    }
+
+    public render() {
+        const styles = this.props.styles;
+        const theme = this.props.theme;
+
+        return (
+            <View style={styles.container}>
+                <Text style={styles.gasPriceLabel}>{translate('Fee.gasPrice')}</Text>
+                <View style={[styles.inputBox, styles.inputBoxTop]}>
+                    <TextInput
+                        testID="gas-price"
+                        style={styles.input}
+                        placeholderTextColor={theme.colors.textSecondary}
+                        placeholder={translate('Fee.gasPrice')}
+                        autoCapitalize={'none'}
+                        autoCorrect={false}
+                        selectionColor={theme.colors.accent}
+                        value={this.state.inputGasPrice}
+                        onChangeText={value => {
+                            this.addGasPrice(value);
+                        }}
+                    />
+                </View>
+
+                <Text style={styles.gasLimitLabel}>{translate('Fee.gasLimit')}</Text>
+                <View style={styles.inputBox}>
+                    <TextInput
+                        testID="gas-limit"
+                        style={styles.input}
+                        placeholderTextColor={theme.colors.textSecondary}
+                        placeholder={translate('Fee.gasLimit')}
+                        autoCapitalize={'none'}
+                        autoCorrect={false}
+                        selectionColor={theme.colors.accent}
+                        value={this.state.inputGasLimit.toString()}
+                        onChangeText={value => {
+                            this.addGasLimit(value);
+                        }}
+                    />
+                </View>
+                <FeeTotal amount={this.props.gasPrice} blockchain={this.props.blockchain} />
```

Comment:
this is weird, `FeeTotal` should display total fee, that is calculated based on `gasPrice` and `gasLimit`